Makefile

```diff
@@ -12,6 +12,9 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# CONTROL_PLANE defines the type of cluster that will be used. Possible values are kubernetes (default) and kcp.
+CONTROL_PLANE ?= kubernetes
```

Comment:
Having kcp off by default makes it easier to overlook when updating the CRD, I'd have it on by default & mandatory.